governance.md

```diff
@@ -0,0 +1,279 @@
+# Main Governance Document
+
+The official version of this document, along with a list of
+individuals and institutions in the roles defined in the governance
+section below, is contained in The Project Governance Repository at:
+
+[https://github.com/pydata/pandas-governance](https://github.com/pydata/pandas-governance)
+
+The Project
+===========
+
+The pandas Project (The Project) is an open source software project affiliated
+with the 501c3 NumFocus Foundation. The goal of The Project is to develop open
+source software and deploy open and public websites and services for
+reproducible, exploratory and interactive computing. The Software developed by
```

Comment:
"reproducible, exploratory and interactive computing" is a leftover from Jupyter ?